jest.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     '<rootDir>/tests/jest-reporters/eslint-check.js',
   ],
   setupFilesAfterEnv: ['<rootDir>/tests/setup.js'],
+  testEnvironment: '<rootDir>/tests/jest-environment.js',
```

Comment:
So we can't "just" use the `node` environment? That's what I understood from https://github.com/jsdom/jsdom/issues/2524#issuecomment-902027138

Reply:
We can't "just" use the `node` environment, because a large number of our tests (those that use `mount`) expect the DOM to be present. I tried switching to the `node` environment, and then requesting the `jsdom` environment just for a test file that needs it, by adding:
```
/**
 * @jest-environment jsdom
 */
```
to the top of the test file, but that just brings back the ` ReferenceError: TextEncoder is not defined` error. 

I think what we need to do, if we want to make this work, is to not request that the `jsdom` environment be used for these tests, but rather to configure each test to use `jsdom`. I haven't been able to figure out how to do that yet, and even if I do figure it out, it seems like a lot to add to each test, and also then we need to be aware of which test files (or cases) need access to `jsdom` which feels a bit ugly to me.

After looking at all of that, I am of the opinion that we should just accept the "hack" as used in this patch, but if you have a suggestion of another way to avoid it I'm happy to give it a try.

Reply:
I agree

Reply:
@willdurand As you agree with the above, and you also agreed with removing the one test that no longer seems to work, I think this is good to merge. Would you be able to officially approve the PR, if you are in agreement?

Reply:
yep, when it isn't failing anymore :)